applications/GeoMechanicsApplication/custom_processes/apply_k0_procedure_process.cpp

```diff
@@ -137,7 +154,9 @@ void ApplyK0ProcedureProcess::CalculateK0Stresses(Element& rElement)
         // Apply K0 procedure
         for (int i_dir = 0; i_dir <= 2; ++i_dir) {
             if (i_dir != k0_main_direction) {
-                rStressVectors[g_point][i_dir] = k0_vector[i_dir] * rStressVectors[g_point][k0_main_direction];
+                rStressVectors[g_point][i_dir] =
+                    k0_vector[i_dir] * (rStressVectors[g_point][k0_main_direction] + POP_value);
+                rStressVectors[g_point][i_dir] -= PoissonURfactor * POP_value;
```

Comment:
Don't know if the formatting is correct, but:
```suggestion
      rStressVectors[g_point][i_dir] =
          k0_vector[i_dir] * (rStressVectors[g_point][k0_main_direction] + POP_value) - PoissonURfactor * POP_value;
```

Reply:
Done